src/Core/Domain/AuthorizationServer/Model/ApiAccessInterface.php

```diff
@@ -67,29 +67,17 @@ public function getClientSecret(): string;
      */
     public function setClientSecret($clientSecret): ApiAccessInterface;
 
-    /**
-     * @return AuthorizedApplicationInterface
-     */
-    public function getAuthorizedApplication(): AuthorizedApplicationInterface;
-
-    /**
-     * @param AuthorizedApplicationInterface $authorizedApplication
-     *
-     * @return $this
-     */
-    public function setAuthorizedApplication(AuthorizedApplicationInterface $authorizedApplication): ApiAccessInterface;
-
     /**
      * @return bool
      */
-    public function isActive(): bool;
+    public function isEnabled(): bool;
 
     /**
      * @param bool $active
      *
      * @return $this
      */
-    public function setActive(bool $active): ApiAccessInterface;
+    public function setEnabled(bool $active): ApiAccessInterface;
```

Comment:
is that possible to rename `$active` to `$enable` ?

Reply:
Indeed, but this interface should be removed later anyway so not blocking